### ISSUE TITLE
Use pika==0.11.0

### ISSUE THREAD

**Issue by akalex** [mosquito/aio-pika#80]
Hello,

I am just curious, why **aio-pika** is restricted to used **pika** with the version less than 0.11.0? Is there something that might broke code functionality? 

Thank you.

**Comment by mosquito** [comment]
Thats because pika/pika has an issue [#875](https://github.com/pika/pika/issues/875).

**Comment by lukebakken** [comment]
Neither this issue nor #54 provides code or information to reproduce what is reported in pika/pika#875. I am trying to get Pika `1.0.0` released and would like to fix this issue, too. Could code that reproduces the issue be provided? Thanks!

**Comment by decaz** [comment]
OMG Pika `1.0.0`. Somebody, please reproduce the issue while we can still jump on the train of progress!

**Comment by mosquito** [comment]
I spent two weeks trying to refactor pika. But I failed. Pika was designed for Python 2.x +, and my refactoring contained so many changes, so it looked like a fork.

`aio-pika> = 5` no longer uses pika and uses aiormq instead.
It was very hard to adapt the code base to the new driver. In fact, from the point of view of the work that was to be done, it was easier to patch the pika's code.

Pamqp is really cool and works great under the hood of aiormq.

I apologize for not making `pika` better, sorry guys.
